### ISSUE TITLE
Problems with char_del_option

### ISSUE THREAD

**Issue by cydh** [rathena/rathena#1259]
````
// Restrict character deletion by email address or birthdate.
// This restricts players from changing the langtype and deleting characters.
// For birthdate, the client must be 20100803 or newer.
// Defaults based on client date.
// 1: Email address
// 2: Birthdate
// 3: Email address or Birthdate
char_del_option: 2
````
Somehow this config that was introduced since 7a9aa7fbcfe40080c1b743137fc68aebb596e632 isn't perfect **because of the client!** (and now it's broken, the checks was removed).
This config will **only work for client that send `0x0068` or `0x01fb` as delete request!** For older clients or client with langtype 1 or 12. (meh, I always use langtype 0 for testing)
As the note in the char_clif source
```c++
			// delete char
			case 0x68: next=chclif_parse_delchar(fd,sd,cmd); break; //
			case 0x1fb: next=chclif_parse_delchar(fd,sd,cmd); break; // 2004-04-19aSakexe+ langtype 12 char deletion packet
```

Newer clients for langtype 0, will use `0x0829` which only has 6 chars for deletion code (is BirthDate), even we force to use email, it only matches for first 6 chars. xD
```c++
			case 0x829: next=chclif_parse_char_delete2_accept(fd, sd); break;
			// CH: <0829>.W <char id>.L <birth date:YYMMDD>.6B
```

And additional info from me, **idRO & iRO** now is using `0x098f` as delete request (I tried change langtype 6 for 2015-11-04aRagexe, still send `0x0829`, so not langtype related).
This packet has varies packet length, notated as
```c++
// CH: <098f>.W <Length>.W <CharId>.L <Code>.?B
```

**It will be nice if someone can make patch for client to force send `0x01fb` for all langtypes.**

**Comment by spinzaku** [comment]
any updates on this @cydh  ? ><

**Comment by aleos89** [comment]
Related to #1299.

**Comment by Lemongrass3110** [comment]
Closing this since it was fixed in 2a1aeba.
